ballista/core/src/client.rs

```diff
@@ -43,16 +43,19 @@ use crate::serde::protobuf;
 use crate::utils::create_grpc_client_connection;
 use datafusion::physical_plan::{RecordBatchStream, SendableRecordBatchStream};
 use futures::{Stream, StreamExt};
-use log::debug;
+use log::{debug, warn};
 use prost::Message;
-use tonic::Streaming;
+use tonic::{Code, Streaming};
 
 /// Client for interacting with Ballista executors.
 #[derive(Clone)]
 pub struct BallistaClient {
     flight_client: FlightServiceClient<tonic::transport::channel::Channel>,
 }
 
+const IO_RETRIES_TIMES: u8 = 3;
+const IO_RETRY_WAIT_TIME_MS: u64 = 3000;
```

Comment:
Better to make these configurations configurable.

Reply:
Will fix this, when support global config.